Validation/RecoTrack/python/TrackValidation_cff.py

```diff
@@ -892,8 +894,10 @@ def _uniqueFirstLayers(layerList):
     dirName = "Tracking/TrackSeeding/",
     label = _seedSelectors,
     doSeedPlots = True,
-    doResolutionPlotsForLabels = [ "seedTracksjetCoreRegionalStepSeeds",]
+    doResolutionPlotsForLabels = [ "seedTracksjetCoreRegionalStepSeedsBarrel","seedTracksjetCoreRegionalStepSeedsEndcap"]
```

Comment:
shouldn't this remain unchanged because there is a replacement applied just a few lines below?
(and without `seedingDeepCore` it seems like `trackValidatorSeedingTrackingOnly` will be broken; or at least will not correspond to what is running in reco)